lib/rmt/cli/repos.rb

```diff
@@ -40,10 +42,12 @@ def clean
     print _('Enter a value:')
     print "\e[22m\s\s"
 
-    input = $stdin.gets.to_s.strip
-    if input != 'yes'
-      puts "\n" + _('Clean cancelled.')
-      return
+    unless options[:no_confirm]
+      input = $stdin.gets.to_s.strip
+      if input != 'yes'
+        puts "\n" + _('Clean cancelled.')
+        return
+      end
```

Comment:
Can we add a console output indicating that the auto confirmation switch was used here?